lib/dor/release/item.rb

```diff
@@ -17,12 +17,12 @@ def object
         @object ||= Dor.find(@druid)
       end
 
-      def object_type
-        unless @obj_type
-          obj_type = object.identityMetadata.objectType
-          @obj_type = (obj_type.nil? ? 'unknown' : obj_type.first)
-        end
-        @obj_type.downcase.strip
+      def cocina_model
+        @cocina_model ||= Dor::Services::Client.object(@druid).find
+      end
+
+      def collection?
```

Comment:
Not a blocking comment: might we later move this method, and related methods (*e.g.*, `#dro?`), into the `cocina-models` gem?